### ISSUE TITLE
Add guidelines on how to create a new lesson on github

### ISSUE THREAD

**Issue by annefou** [coderefinery/manuals#13]
Lesson design is very nice but some information on how to create a new lesson (not to design it) is missing. 

Does it make sense to add a section on how to create a CodeRefinery lesson (with CodeRefinery lesson template, etc.). I would be interested because so far I only use Carpentries lesson template.

**Comment by bast** [comment]
I agree, this is needed. In particular the `jekyll-common` part is not well documented I think.
